src/Features/Core/Portable/SignatureHelp/CommonSignatureHelpService.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.SignatureHelp
+{
+    internal abstract class CommonSignatureHelpService : SignatureHelpServiceWithProviders
```

Comment:
Not yet. I modeled this off of CommonCompletionService. If I find I don't need it, I'll get rid of it.